pkg/cmd/admin/registry/registry.go

```diff
@@ -426,6 +428,18 @@ func (opts *RegistryOptions) RunCmdRegistry() error {
 				},
 			},
 		})
+	} else if opts.Config.Deployment {
```

Comment:
I guess we could provide an option for Deployments but we cannot move away from DCs yet.